app/graphql/mutations/update_project.rb

```diff
@@ -0,0 +1,13 @@
+module Mutations
+  class UpdateProject < BaseMutation
+    argument :input, Types::Inputs::UpdateProjectInput, required: true
+
+    type Types::Payloads::ProjectPayload
+
+    def resolve(input:)
+      result = Projects::Update.call(project: ::Project.find_by(id: input.id), project_params: input.to_h.except(:id))
+
+      result.to_h.merge(errors: formatted_errors(result.project))
+    end
```

Comment:
We can do here some refactoring, and please check that is works correctly

```suggestion
    def resolve(input:)
      result = Projects::Update.call(project: :project, project_params: input.to_h)

      result.to_h.merge(errors: formatted_errors(result.project))
    end
    
   private
   
   def project
     ::Project.find_by(id: input.delete(:id))
   end
```